backends/verilog/verilog_backend.cc

```diff
@@ -1605,6 +1605,29 @@ void dump_process(std::ostream &f, std::string indent, RTLIL::Process *proc, boo
 	}
 }
 
+void dump_parameters(std::ostream &f, std::string indent, RTLIL::Module *module)
+{
+	for (auto& param : module->avail_parameters) {
+
+		// Skip the parameter if its name begins with '$'. Such names are illegal
```

Comment:
I don't believe this is correct?  You can use escape identifiers to write them out?

Reply:
Fixed.